### ISSUE TITLE
[BUG] carbonapi Debian package uses incorrect systemd EnvironmentFile

### ISSUE THREAD

**Issue by ryangsteele** [go-graphite/carbonapi#551]
**Describe the bug**
The latest stable carbonapi Debian package deploys two versions of the same environment file: `/etc/sysconfig/carbonapi` and `/etc/default/carbonapi`. While `/etc/sysconfig` is the canonical location for such files for RHEL and its derivatives, [Debian-based systems should use `/etc/default`](https://www.debian.org/doc/manuals/maint-guide/dother.en.html#initd) as the location for environment files sourced by systemd/sysvinit/etc. scripts. Additionally, the systemd config points at the version in `/etc/sysconfig` instead of the one in `/etc/default`.

TL;DR: The deb packages should not deploy `/etc/sysconfig/carbonapi`, and `/etc/systemd/system/carbonapi.service` should load the `EnvironmentFile` from `/etc/default/carbonapi`.

I'll happily submit a patch that fixes this if you can point me to the repository containing the deb packaging configs (I looked around, but was unable to locate them).

**CarbonAPI Version**
0.14.1-1 (latest stable deb available from [packagecloud.io](https://packagecloud.io/go-graphite/stable?filter=debs))

**Logs**
N/A

**CarbonAPI Configuration:**
N/A

**Simplified query (if applicable)**
N/A

**Backend metric retention and aggregation schemas**
N/A

**Backend response (if possible)**
N/A

**Additional context**

```
$ dpkg -l carbonapi
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                          Version             Architecture        Description
+++-=============================-===================-===================-===============================================================
ii  carbonapi                     0.14.1-1            amd64               carbonapi: replacement for graphite API server
```
```
$ dpkg -L carbonapi 
/etc
/etc/carbonapi
/usr
/usr/bin
/usr/bin/carbonapi
/usr/share
/usr/share/carbonapi
/usr/share/carbonapi/carbonapi.example.yaml
/etc/sysconfig
/etc/sysconfig/carbonapi
/etc/systemd
/etc/systemd/system
/etc/systemd/system/carbonapi.service
/etc/default
/etc/default/carbonapi
/etc/logrotate.d
/etc/logrotate.d/carbonapi
```
```
$ grep EnvironmentFile /etc/systemd/system/carbonapi.service
EnvironmentFile=/etc/sysconfig/carbonapi
```
```
$ ls -l /etc/sysconfig
total 4
-rw-r--r-- 1 root root 55 Aug 19 10:07 carbonapi
```

**Comment by Civil** [comment]
Yeah, that's an artifact of my latest attempt to fix building packages for everything.

Actual scripts are there (for example):
https://github.com/go-graphite/helper-scripts/blob/main/docker/debian-buster/create_package_deb.sh

But I'm not sure if that's the latest version of those scripts... (it should be the same, but I was thinking about rewriting everything yet again and migrate away from travis and rethink how docker images are used at this moment)

**Comment by ryangsteele** [comment]
Thanks! I'll take a look at it as soon as I have some free time.

**Comment by Civil** [comment]
That should be fixed in `carbonapi-0.15.2~57+sha.gbb33f2`

Please reopen if that still doesn't work.
